devicon.json

```diff
@@ -2156,6 +2156,15 @@
             }
         ]
     },
+    {
+        "name": "phoenix",
+        "tags": ["framework", "elixir"],
+        "versions": {
+            "svg": ["original", "plain"]
```

Comment:
A `font` version is missing for `phoenix`. We need at least one font version to add the icon to our set :)